presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetColumnChunkPageReader.java

```diff
@@ -13,35 +13,36 @@
  */
 package com.facebook.presto.hive.parquet.reader;
 
-import com.facebook.presto.hive.parquet.ParquetCodecFactory.BytesDecompressor;
-import com.google.common.primitives.Ints;
-import parquet.column.page.DataPage;
-import parquet.column.page.DataPageV1;
-import parquet.column.page.DataPageV2;
-import parquet.column.page.DictionaryPage;
-import parquet.column.page.PageReader;
+import com.facebook.presto.hive.parquet.ParquetDataPage;
+import com.facebook.presto.hive.parquet.ParquetDataPageV1;
+import com.facebook.presto.hive.parquet.ParquetDataPageV2;
+import com.facebook.presto.hive.parquet.ParquetDictionaryPage;
+import com.facebook.presto.hive.parquet.ParquetPageReader;
+import parquet.hadoop.metadata.CompressionCodecName;
 
 import java.io.IOException;
 import java.util.LinkedList;
 import java.util.List;
 
+import static com.facebook.presto.hive.parquet.ParquetCompressUtils.decompress;
+
 class ParquetColumnChunkPageReader
```

Comment:
I find the name of this class confusing because the name `ParquetColumnChunkPageReader` has both `ColumnChunk` and `Page` in it, which are two separate entities in Parquet. And as far as I understand this only reads pages, so how about renaming this class to `ParquetPageReader` and get rid of the existing `ParquetPageReader` interface, which only has this class as its implementation? Do you envision having other implementations for the existing `ParquetPageReader` interface?